module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+        Post - submits data to a specified resource
+        Get - retrieves data from a specified resource
+        Put - updates a resource completely through a specified resource
+        Delete - deletes a specified resource
+        I believe the fifth verb is 'Patch' and it does the same thing as 'Put'.

```

Comment:
🎉 Some people will tell you that PATCH is supposed to be used when you're changing only *part* of a resource (vs. PUT to replace the whole thing), but that generally hasn't come into play.